### DISCUSSION TITLE
MediaCMS image on DockerHub?

### DISCUSSION THREAD

**Discussion by Dan1ell** [mediacms-io/mediacms#65] [Q&A] [answered]
Hey, so is this image on DockerHub yours?

https://hub.docker.com/r/duffleman/mediacms

Or is Duffelman just some random person?

**Comment by swiftugandan** [comment]
We have not yet started maintaining an image on dockerhub, @mgogoulos right?

**Comment by mgogoulos** [comment]
Not aware of it, just saw it. Makes sense to contact @Duffleman and ask how often this is maintained. I've also have an account info@mediacms.io on hub.docker.com that could be used to upload a mediacms image and make sure it gets maintained

**Comment by mgogoulos** [comment, marked as answer]
It's there - https://hub.docker.com/repository/docker/mediacms/mediacms

**Reply by Dan1ell** [reply]
Could you update the installation instructions with commands for dockerhub?
I just tried "docker run -d mediacms/mediacms" but it exits. In the log I see:
django.db.utils.OperationalError: could not translate host name "db" to address: Name or service not known

**Reply by swiftugandan** [reply]
This image cannot be used by itself. It depends on an external postgres and redis instance. (eg. Container ... you can set these in deploy/docker/local_settings.py)

See this [documentation](https://github.com/mediacms-io/mediacms/blob/main/docs/Docker_deployment.md)

We are trying to improve the docs, if it is not clear, please highlight areas we might improve.

**Reply by mgogoulos** [reply]
I've just updated the instructions, so that the Docker Hub image will be used and no `docker-compose build` is needed anymore
